### DISCUSSION TITLE
Is is possible to export only annotated images when exporting a dataset?

### DISCUSSION THREAD

**Discussion by cyrilzakka** [cvat-ai/cvat#2837] [Q&A] [answered]
I have a dataset of around 16GB. For now, only around 600 images are annotated but exporting the dataset exports all of the images including the ones not annotated. Is is possible to export the dataset with only the annotated images?

**Comment by bsekachev** [comment, marked as answer]
As far as I remember it is not possible.
@azhavoro Please, correct me if I am wrong.

**Reply by narayan1234chalo** [reply]
May you answer my question please how can I get unique images from annotation?

**Comment by nirabo** [comment]
The same need I had in cherry-picking annotated frames of interest.

What I ended up doing:
0. Create a new task and upload images
1. Annotate only frames of interest
2. Export the complete dataset
3. Implement a short script that crawls the exported data and splits it to annotated and non-annotated frames/annotations
4. Re-upload the annotated and non-annotated sets as new tasks
5. Remove the initial task

I know that's slow, dumb and cumbersome, but that is my initial solution. I am now looking into the SDK/API and I think there's plenty of degrees of freedom to get it done more streamlined there, however this still will be a viable extension to CVAT UI.

**Comment by okadapau2** [comment]
yes it is possible,
Filter Annotated Images: In the CVAT interface, navigate to the project containing your dataset. Use the filter or search functionality to display only the images that have been annotated. This will typically involve applying a filter based on annotation status or presence.

Select Annotated Images: Once you've filtered the images to display only the annotated ones, select all of them. This may involve selecting them individually or using a "select all" option if available.

Export Selected Images: After selecting the annotated images, look for an option to export them. CVAT typically provides functionality for exporting selected images or annotations. Depending on your requirements, you may have the option to export the images along with their annotations in a specific format (e.g., Pascal VOC, COCO, YOLO) or as standalone images with annotation files.

Choose Export Format and Options: When exporting the selected images, choose the desired export format and any relevant options. For example, you may need to specify the output directory, file naming conventions, or annotation format settings. Make sure to review and adjust the export options according to your preferences and requirements.

Initiate Export: Once you've configured the export settings, initiate the export process. Depending on the size of the dataset and the selected export format, this may take some time to complete. Monitor the progress of the export process and wait for it to finish.

Verify Exported Data: After the export process is complete, verify that the exported dataset contains only the annotated images. Check the exported files to ensure that they include the desired annotations and meet your quality standards.

**Reply by okadapau2** [reply]
its very possible, you can follow this procedure

Filter Annotated Images: In the CVAT interface, navigate to the project containing your dataset. Use the filter or search functionality to display only the images that have been annotated. This will typically involve applying a filter based on annotation status or presence.

Select Annotated Images: Once you've filtered the images to display only the annotated ones, select all of them. This may involve selecting them individually or using a "select all" option if available.

Export Selected Images: After selecting the annotated images, look for an option to export them. CVAT typically provides functionality for exporting selected images or annotations. Depending on your requirements, you may have the option to export the images along with their annotations in a specific format (e.g., Pascal VOC, COCO, YOLO) or as standalone images with annotation files.

Choose Export Format and Options: When exporting the selected images, choose the desired export format and any relevant options. For example, you may need to specify the output directory, file naming conventions, or annotation format settings. Make sure to review and adjust the export options according to your preferences and requirements.

Initiate Export: Once you've configured the export settings, initiate the export process. Depending on the size of the dataset and the selected export format, this may take some time to complete. Monitor the progress of the export process and wait for it to finish.

Verify Exported Data: After the export process is complete, verify that the exported dataset contains only the annotated images. Check the exported files to ensure that they include the desired annotations and meet your quality standards.

**Reply by ol-loginov** [reply]
Where is that "filter or search functionality to display only the images that have been annotated" on project page?

It's unclear how to "display only the images that have been annotated" - there is no such option in "filters" dropdown
